crd/multitenancy/api/v1alpha1/nodeinfo.go

```diff
@@ -36,6 +38,17 @@ type NodeInfoSpec struct {
 	VMUniqueID string `json:"vmUniqueID,omitempty"`
 }
 
+// NodeInfoStatus defines the observed state of NodeInfo
+type NodeInfoStatus struct {
+	// +kubebuilder:validation:Optional
+	DeviceInfos []DeviceInfo `json:"deviceInfos,omitempty"`
+}
+
+type DeviceInfo struct {
+	DeviceType  DeviceType `json:"deviceType,omitempty"`
+	DeviceCount int        `json:"deviceCount"`
```

Comment:
is this expected count of device type x?

Reply:
yes. For device plugin implementation, we need count of device types.